main.go

```diff
@@ -397,3 +403,13 @@ func main() {
 	}
 	// ---
 }
+
+// getStartTime gets the start time of the build.
+func getStartTime() (time.Time, error) {
+	startTimeString := os.Getenv("BITRISE_BUILD_TRIGGER_TIMESTAMP")
```

Comment:
why did do you this change? what if there are multiple xamarin-archive steps in the workflow. If the last one runs, but generates 0 artifacts, the step will find the previous artifacts.

Reply:
I haven't thought about this case, but the reason is the following: if there are multiple steps in a workflow, one generates for iOS, one for macOS, the app folder won't be found for the macOS, as the iOS build already generated it (and the mod time will be older). This is why I updated it to the start of the build, but your case could be in issue as well, so have to come up with different solution.